tests/Faithlife.Analyzers.Tests/Helpers/DiagnosticVerifier.Helper.cs

```diff
@@ -58,7 +69,10 @@ protected static Diagnostic[] GetSortedDiagnosticsFromDocuments(DiagnosticAnalyz
 			var diagnostics = new List<Diagnostic>();
 			foreach (var project in projects)
 			{
-				var compilationWithAnalyzers = project.GetCompilationAsync().Result.WithAnalyzers(ImmutableArray.Create(analyzer));
+				var compilation = project.GetCompilationAsync().Result;
```

Comment:
Pre-existing bug, but should prefer `.GetAwaiter().GetResult()`.

Reply:
6504db14b1a934e7ec05c5eeef771e9ea8edd594